classes/Book/label.rb

```diff
@@ -0,0 +1,15 @@
+class Label
+  attr_accessor :items, :color, :title, :id
+
+  def initialize(color, title, id = nil)
+    @id = id || Random.rand(1..1000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
```

Comment:
I would highly recommend you to add the `item` with a conditional
  ```rb
    @items << item unless @items.include?(item)
  ``` 
to avoid adding the same instance to the collection of `items`.